sdk/python/the0/state.py

```
@@ -0,0 +1,182 @@
+
+
+def _get_key_path(key: str) -> str:
+    """Get the file path for a state key."""
+    state_dir = _get_state_dir()
+    return os.path.join(state_dir, f"{key}.json")
+
+
+def _validate_key(key: str) -> None:
+    """Validate that a key is safe to use as a filename."""
+    if not key:
+        raise ValueError("State key cannot be empty")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)